.github/workflows/windows.yml

```diff
@@ -111,16 +34,16 @@ jobs:
           PATH: '${env:PATH};C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\amd64'
         run: |
           cd vs
-          MSBuild -m dosbox.sln -p:Configuration=${{ matrix.type }} | Tee-Object build.log
+          MSBuild -m dosbox.sln -p:Configuration=Debug -p:Platform=${{ matrix.conf.vs-platform }} | Tee-Object build.log
       - name:  Summarize warnings
         shell: pwsh
         env:
-          MAX_WARNINGS: 348
+          MAX_WARNINGS: ${{ matrix.conf.max_warnings }}
         run: python scripts\count-warnings.py --msvc vs\build.log
 
 
   build_windows_vs_release:
-    name: Release build
+    name: Release build (32-bit)
```

Comment:
Nice differentiation. 
This is out of scope, but it won't be much of a jump to fold this `build_windows_vs_release:` up into the matrix  and create 32-bit and 64-bit releases.

Reply:
Yup. I decided to skip it for now, so we can verify if all the right files are picked for 64-bit MSVC compilation (to avoid the same problem we had with msys2 builds at one point).